test/verify/check-image

```diff
@@ -192,6 +192,85 @@ class TestImage(composerlib.ComposerCase):
         # collect code coverage result
         self.check_coverage()
 
+    def testOSTree(self):
```

Comment:
Can we skip this test on rhel8 and fedora-31? It's more reasonable and efficient. Thanks
```python
@unittest.skipUnless(os.environ.get("TEST_OS") == "fedora-32", "Do not support ostree image")
```
And we can changed to `skipIf` when ostree image lands on rhel8
```python
@unittest.skipIf(os.environ.get("TEST_OS") == "fedora-31", "Do not support ostree image")
```
Please `import os` and `import unittest`.

Reply:
I added this. Thanks!